easybuild/easyconfigs/z/zarr/zarr-3.0.9-foss-2025a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = "PythonBundle"
+
+name = 'zarr'
+version = '3.0.9'
```

Comment:
so 3.0.10 is also out, though I couldn't tell if the 2 bug fixes are high impact or not https://zarr.readthedocs.io/en/stable/release-notes.html#id1